it/controllers/propertyLinking/UploadControllerISpec.scala

```diff
@@ -0,0 +1,1417 @@
+package controllers.propertyLinking
+
+import base.ISpecBase
+import binders.propertylinks.ClaimPropertyReturnToPage
+import com.github.tomakehurst.wiremock.client.WireMock._
+import models.upscan._
+import models._
+import org.jsoup.Jsoup
+import play.api.http.HeaderNames
+import play.api.http.Status._
+import play.api.libs.json.{JsObject, JsString, Json}
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import repositories.PropertyLinkingSessionRepository
+import uk.gov.hmrc.http.{HeaderCarrier, SessionId}
+
+import java.time.LocalDate
+import java.util.UUID
+
+class UploadControllerISpec extends ISpecBase {
+
+  val testSessionId = s"stubbed-${UUID.randomUUID}"
+
+  lazy val mockPropertyLinkingSessionRepository: PropertyLinkingSessionRepository = app.injector.instanceOf[PropertyLinkingSessionRepository]
+  implicit val hc: HeaderCarrier = HeaderCarrier(sessionId = Some(SessionId(testSessionId)))
+
+  // Business rates bill page text
+  val ratesBillHeadingText = "Upload your business rates bill"
+  val ratesBillHeadingTextWelsh = "Lanlwytho eich bil ardrethi busnes"
+  val ratesBillHeadingTextAgent = "Upload your client’s business rates bill"
+  val ratesBillHeadingTextAgentWelsh = "Lanlwytho bil ardrethi busnes eich cleient"
+
+  // Lease page text
+  val leaseHeadingText = "Upload your lease"
+  val leaseHeadingTextWelsh = "Lanlwytho eich prydles"
+  val leaseHeadingTextAgent = "Upload your client’s lease"
+  val leaseHeadingTextAgentWelsh = "Lanlwytho prydles eich cleient"
+
+  // Licence to occupy page text
+  val licenceHeadingText = "Upload your licence to occupy"
+  val licenceHeadingTextWelsh = "Lanlwytho eich trwydded i feddiannu"
+  val licenceHeadingTextAgent = "Upload your client’s licence to occupy"
+  val licenceHeadingTextAgentWelsh = "Lanlwytho trwydded eich cleient i feddiannu"
+
+  // Service charge statement
+  val serviceChargeHeadingText = "Upload your service charge statement"
+  val serviceChargeHeadingTextWelsh = "Lanlwytho eich datganiad tâl gwasanaeth"
+  val serviceChargeHeadingTextAgent = "Upload your service charge statement"
```

Comment:
Is there a reason why for Service charge, Stamp duty, Land registry, Water rate and Utility all don't have a client variation of the text, whereas Business rates bill, license and lease do?

Reply:
I'm not sure why that is. I raised [VTCCA-6490](https://jira.tools.tax.service.gov.uk/browse/VTCCA-6490) to fix this. I added different vals at the top to make it easy to update the tests when this change is made